opendcs-rest-api/build.gradle

```diff
@@ -113,14 +113,16 @@ sonarqube {
 
 
 // task to generate OpenAPI JSON file
-resolve {
-    classpath = sourceSets.main.runtimeClasspath
-    outputFileName = 'opendcs-openapi'
-    outputFormat = providers.gradleProperty('outputFormat').getOrElse('JSON')
-    prettyPrint = 'TRUE'
-    resourcePackages = ['org.opendcs.odcsapi.res', 'org.opendcs.odcsapi.beans', 'org.opendcs.odcsapi.sec']
-    outputDir = file('build/swagger/')
-}
+//resolve {
```

Comment:
What broke that made you comment this out?

That such happened doesn't surprised me, there was some back api output that's fixed in the #550 PR.